sdk/communication/azure-communication-common/src/main/java/com/azure/communication/common/CommunicationTokenRefreshOptions.java

```diff
@@ -22,23 +22,23 @@
     public CommunicationTokenRefreshOptions(TokenRefresher tokenRefresher, boolean refreshProactively) {
         this.tokenRefresher = tokenRefresher;
         this.refreshProactively = refreshProactively;
-        this.token = null;
+        this.initialToken = null;
     }
 
      /**
      * Creates a CommunicationTokenRefreshOptions object
-     * 
-     * @param tokenRefresher the token refresher to provide capacity to fetch fresh token
+     *
+     * @param tokenRefresher the initialToken refresher to provide capacity to fetch fresh initialToken
      * @param refreshProactively when set to true, turn on proactive fetching to call
-     *                           tokenRefresher before token expiry by minutes set
+     *                           tokenRefresher before initialToken expiry by minutes set
      *                           with setCallbackOffsetMinutes or default value of
      *                           two minutes
-     * @param token the optional serialized JWT token
+     * @param initialToken the optional serialized JWT initialToken
      */
-    public CommunicationTokenRefreshOptions(TokenRefresher tokenRefresher, boolean refreshProactively, String token) {
+    public CommunicationTokenRefreshOptions(TokenRefresher tokenRefresher, boolean refreshProactively, String initialToken) {
```

Comment:
Same, let's stick with `token`

Reply:
The other `ctor` is static token so we call it `token` but this one is just the initial one, so we should call it `initialToken`